examples/websockets/assets/script.js

```diff
@@ -7,3 +7,23 @@ socket.addEventListener('open', function (event) {
 socket.addEventListener('message', function (event) {
     console.log('Message from server ', event.data);
 });
+
+window.addEventListener("load", (event) => {
+   console.log("page is fully loaded");
+
+});
```

Comment:
What is this for?

Reply:
Well following your logic that examples must be about axum only this serves no purpose at the moment. I'll trim it.

Reply:
> examples must be about axum only

I didn't say that. I said they should be related to axum. For example a while ago someone submitted a PR with an sqlx example that included a whole setup for migrations. But migrations aren't related to axum so they don't need to be in the example.

I think there is value in being able to open the browser and see data coming in from the websocket. Having a 9 line JS file for that is totally fine.

Reply:
Exactly, and that message in particular was pure bloat, just on a smaller scale.